### PR TITLE
UI: go to first/last page buttons in pagination

### LINKED ISSUE #604: [Improvement request]: Go to first / last page buttons in Pagination components
### Contact Details

jurre@shelf.nu

### Is your feature request related to a problem? Please describe?

If a user has a large amount of assets, or other entries in other tables, they'll have to click the next button multiple times to reach the last page. 

### Describe the solution you'd like

Adding 'Go to first / last page' buttons within the Pagination component used throughout multiple tables within the app. In this new designs I have also changed the icons used in the buttons. Using Chevrons instead Arrow. Thanks!

- [ ] Change Arrow icons used for navigating to next / previous page into Chevron icons. Chevron icons are already being used within the app so please use those to replace Arrow icons
- [ ] Add 'Go to last / first page buttons. These buttons should use the Double chevron buttons. I'm providing these icons down below. Please make sure the icons use the same conventions (colors, container, etc.) as other icons used throughout the app. I am providing only 1 SVG file. Please use CSS to flip the icon.

[chevron-left-double.svg.zip](https://github.com/Shelf-nu/shelf.nu/files/13672697/chevron-left-double.svg.zip)

DESIGNS

Screenshot 2023-12-14 at 12 10 40

Screenshot 2023-12-14 at 12 10 57

### Describe alternatives you've considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by GauravBurande**
closes #604 
/claim #604

image

### CODE REVIEW

File: app/components/icons/library.tsx [hunk 1]
```diff
@@ -929,6 +929,25 @@ export const ArrowLeftIcon = (props: SVGProps<SVGSVGElement>) => (
...
+    width="20"
+    height="20"
+    viewBox="0 0 20 20"
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    {...props}
+  >
+    <path
+      d="M15 14.1666L10.8333 9.99998L15 5.83331M9.16667 14.1666L5 9.99998L9.16667 5.83331"
+      stroke="#667085"
```
Comment (DonKoko) [inline app/components/icons/library.tsx]: replace with `currentColor`

File: app/components/list/pagination/pagination.tsx [hunk 2]
```diff
@@ -57,6 +77,19 @@ export const Pagination = () => {
...
         </Button>
+
+        <Button
+          variant="secondary"
+          size="sm"
+          to={lastPage}
+          disabled={prevDisabled}
+          className={tw(
+            "rotate-180 rounded-none border-y-0 border-l-0 border-r border-gray-300 bg-transparent px-4 py-3 hover:bg-transparent",
+            prevDisabled && "opacity-50"
```
Comment (DonKoko) [inline app/components/list/pagination/pagination.tsx]: This should be `nextDisabled`

**Review by DonKoko — CHANGES_REQUESTED** [review]
Okey so I added a few comments in the code and I have some general comments:
Screenshot 2024-02-01 at 14 11 59

- [x] The prev/next arrows are not changed to singular chevrons(see issue, there its explained). You have to use the component <ChevronRight>
- [x] The height of those buttons dont match the height of the page selector. Make sure they fit properly

**Comment by GauravBurande** [comment]
> Okey so I added a few comments in the code and I have some general comments: Screenshot 2024-02-01 at 14 11 59
> 
> * [ ]  The prev/next arrows are not changed to singular chevrons(see issue, there its explained). You have to use the component
> * [ ]  The height of those buttons dont match the height of the page selector. Make sure they fit properly

fixed. The height is matched and replaced arrows with chevrons

**Review by DonKoko — APPROVED** [review]

**Pull request merged**